pkgs/by-name/da/daggerfall-unity/package.nix

```diff
@@ -0,0 +1,110 @@
+{
+  alsa-lib,
+  autoPatchelfHook,
+  copyDesktopItems,
+  fetchzip,
+  lib,
+  libGL,
+  libXScrnSaver,
+  libXcursor,
+  libXi,
+  libXinerama,
+  libXrandr,
+  libXxf86vm,
+  libpulseaudio,
+  libudev0-shim,
+  makeDesktopItem,
+  nix-update-script,
+  stdenv,
+  vulkan-loader,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "daggerfall-unity";
+  version = "1.1.1";
+
+  src = fetchzip {
+    url = "https://github.com/Interkarma/daggerfall-unity/releases/download/v${finalAttrs.version}/dfu_linux_64bit-v${finalAttrs.version}.zip";
+    stripRoot = false;
+    hash = "sha256-JuhhVLpREM9e9UtlDttvFUhHWpH7Sh79OEo1OM4ggKA=";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  buildInputs = [
+    alsa-lib
+    copyDesktopItems
```

Comment:
This needs to be in `buildInputs`? I'm asking because I genuinely don't know, I'd expected it to be a build-time thing, not run-time.

Reply:
Then please move `copyDesktopItems` to `nativeBuildInputs`, maybe you can catch more with `strictDeps = true`, otherwise the derivation dependents look good!

Reply:
Thanks! Is `strictDeps` documented anywhere? It seems like it should, but there's nothing in https://nixos.org/manual/nixpkgs/stable/.

Reply:
Huh, I could've sworn it was documented there. Either way, `strictDeps` tries to enforce the dependency resolution "stricter", by making derivations available only where and when they should, which can cause quick failures for misrepresented dependencies.